ingest/models.py

```diff
@@ -259,4 +263,55 @@ def __str__(self):
     r24_name = models.CharField(max_length=256)
     r24_directory = models.CharField(max_length=256)
     
+class BilUser(models.Model):
+
+    name = models.CharField(max_length=256)
+    orcid = models.CharField(max_length=256)
+    affiliation = models.CharField(max_length=256)
+    affiliation_identifier = models.CharField(max_length=256)
+    auth_user_id = models.ForeignKey(auth_user, on_delete=models.SET_NULL, null = True, blank=True)
+
+class Project(models.Model):
+    
+    project_pi = models.ForeignKey(User, on_delete=models.SET_NULL, null=True, blank=True)
+    funded_by = models.CharField(max_length=256)
+    is_biccn = models.BooleanField(default=False)
+    
+class ProjectCollections(models.Model):
+    def __str__(self):
+        return self.name
+    project_id = models.ForeignKey(Project, on_delete=models.SET_NULL, null=True, blank=True)
+    collection_id = models.ForeignKey(Collection, on_delete=models.SET_NULL, null=True, blank=True)
+
+class ProjectUsers(models.Model):
+ 
+    project_id = models.ForeignKey(Project, on_delete=models.SET_NULL, null=True, blank=True)
+    bil_user_id = models.ForeignKey(BilUser, on_delete=models.SET_NULL, null = True, blank=True)
+    is_pi = models.BooleanField(default=False)
+    is_po = models.BooleanField(default=False)
+    role = models.CharField(max_length=256)
+
+class Funder(models.Model)
+
+    name = models.CharField(max_length=256)
+    funding_reference_identifier = models.CharField(max_length=256)
+    funding_reference_identifier_type = models.CharField(max_length=256)
+    award_number = models.CharField(max_length=256)
+    award_title = models.CharField(max_length=256)
+    grant_number = models.CharField(max_length=256)
+
+class ProjectFunders(models.Model):
+
+    project_id = models.ForeignKey(Project, on_delete=models.SET_NULL, null=True, blank=True)
+    funder_id = models.ForeignKey(Funder, on_delete=models.SET_NULL, null=True, blank=True)
 
+class EventsLog(models.Model)
+    collection_id = models.ForeignKey(Collection, on_delete=models.SET_NULL, null=True, blank=True)
```

Comment:
on_delete=models.SET_NULL would be useful here to preserve context historically within this EventsLog model